src/os/bluestore/BlueStore.cc

```diff
@@ -3760,6 +3760,10 @@ int BlueStore::_open_alloc()
   alloc = Allocator::create(cct, cct->_conf->bluestore_allocator,
                             bdev->get_size(),
                             min_alloc_size);
+  if (alloc == nullptr)
+    lderr(cct) << "Allocator::create unknown alloc type" << cct->_conf->bluestore_allocator << dendl;
+    return -EINVAL;
```

Comment:
Or we should not allow any null value to bluestore_allocator or we should check only valid ones are entered in the conf file. That check better be before open_alloc()